ee/codegen/src/__test__/nodes/__snapshots__/map-node.test.ts.snap

```diff
@@ -46,6 +46,19 @@ class MapNode(BaseMapNode):
 "
 `;
 
+exports[`MapNode > basic > map node inputs file 1`] = `
+"from typing import Any, Optional, Union
+
+from vellum.workflows.inputs import BaseInputs
+
+
+class Inputs(BaseInputs):
+    items: Optional[Any] = None
```

Comment:
Added a codegen snapshot to highlight why we needed this PR. In the future, we should be able to generate this:

```
from vellum.workflows.nodes import MapNode

class Inputs(MapNode.SubworkflowInputs):
    pass
```